app/helpers/storage_resource_helper/textual_summary.rb

```diff
@@ -35,6 +39,14 @@ def textual_logical_total
     {:label => _("Total Size"), :value => number_to_human_size(@record.logical_total, :precision => 2)}
   end
 
+  def textual_ems
+    textual_link(@record.ext_management_system)
+  end
+
+  def textual_physical_storage
+    @record.physical_storage
```

Comment:
Stupid question (non-UI dev here) but should this be a textual_link to the physical_storage?  Or are you showing properties from the physical_storage record directly here?

Reply:
I guess it should be a textual_link, though it actually works as a link anyway

Reply:
That is the way it is in the HostInitiatorHelper:
```
  def textual_physical_storage
    textual_link(@record.physical_storage)
  end
```

So yes I think it would be best to stick with that

Reply:
done.
but what value does this method add? just out of curiosity..

Reply:
It handles showing a link to the argument passed in.